DPGAnalysis/Skims/python/ZElectronSkim_cff.py

```diff
@@ -1,13 +1,10 @@
 import FWCore.ParameterSet.Config as cms
-
 # run on MIONAOD
 RUN_ON_MINIAOD = False
-#print "ZEE SKIM. RUN ON MINIAOD = ",RUN_ON_MINIAOD
+print "ZEE SKIM. RUN ON MINIAOD = ",RUN_ON_MINIAOD
```

Comment:
it looks like this printout has introduced problems to T0
https://its.cern.ch/jira/browse/CMSTZ-480
@fabiocos

Reply:
I've removed the printout. Should I submit a new PR?